src/shared/views/Mapper/Tile.js

```diff
@@ -143,8 +151,14 @@ export class _Tile extends React.Component<Props> {
         return { uri: tile.url };
     }
 
+    getOsmBuildings = () => {
+        const { tile } = this.props;
+        return { uri: tile.urlB };
+    }
+
     zoomRender = () => {
         const imageSource = this.getImgSource();
+        const osmBuiling = this.getOsmBuildings();
```

Comment:
Typo: `osmBuiling` -> `osmBuildings` ? Maybe worth renaming it into something like `osmBuildingsImageSource`?